### ISSUE TITLE
which classifier is best for multiclass( say 10) image classification?

### ISSUE THREAD

**Issue by tejasri19** [anujshah1003/Transfer-Learning-in-keras---custom-data#6]
why did u chose softmax function ?
